src/main/kotlin/org/koitharu/kotatsu/parsers/site/vi/TruyenGG.kt

```diff
@@ -0,0 +1,243 @@
+		val url = when {
+			!filter.query.isNullOrEmpty() -> {
+				buildString {
+					append("https://")
+					append(domain)
+					append("/tim-kiem/trang-$page.html")
+					append("?q=")
+					append(filter.query.urlEncoded())
+				}
+			}
+			else -> {
+				buildString {
+					append("https://")
+					append(domain)
+					append("/tim-kiem-nang-cao/trang-")
+					append(page.toString())
+					append(".html?country=")
+
+					if (filter.types.isNotEmpty()) {
+						filter.types.oneOrThrowIfMany()?.let {
+							append(
+								when (it) {
+									ContentType.MANHUA -> '1'
+									ContentType.OTHER -> '4' // Việt Nam
+									ContentType.MANHWA -> '2'
+									ContentType.MANGA -> '3'
+									ContentType.COMICS -> '5'
+									else -> '0' // all
+								},
+							)
+						}
+					} else append('0')
+
+					append("&status=")
+					if (filter.states.isNotEmpty()) {
+						filter.states.oneOrThrowIfMany()?.let {
+							append(
+								when (it) {
+									MangaState.ONGOING -> '0'
+									MangaState.FINISHED -> '1'
+									else -> "-1"
+								},
+							)
+						}
+					} else {
+						append("-1")
+					}
+
+					append("&category=")
+					append(filter.tags.joinToString(separator = ",") { it.key })
+
+					append("&notcategory=")
+					append(filter.tagsExclude.joinToString(separator = ",") { it.key })
+
+					append("&minchapter=0")
+
+					append("&sort=")
+					append(
+						when (order) {
+							SortOrder.NEWEST -> "0"
+							SortOrder.NEWEST_ASC -> "1"
+							SortOrder.UPDATED -> "2"
+							SortOrder.UPDATED_ASC -> "3"
+							SortOrder.POPULARITY -> "4"
+							SortOrder.POPULARITY_ASC -> "5"
+							else -> "2"
+						},
+					)
+				}
+			}
+		}
+
+		val doc = webClient.httpGet(url).parseHtml()
+		return doc.select(".list_item_home .item_home").map { div ->
+			val href = div.selectFirstOrThrow("a.book_name").attrAsRelativeUrl("href")
+			Manga(
+				id = generateUid(href),
+				title = div.select("a.book_name").text(),
+				altTitle = null,
+				url = href,
+				publicUrl = href.toAbsoluteUrl(domain),
+				rating = RATING_UNKNOWN,
+				isNsfw = isNsfwSource,
+				coverUrl = div.selectFirst(".image-cover img")?.attr("data-src").orEmpty(),
+				tags = emptySet(),
+				state = null,
+				author = null,
+				source = source,
+			)
+		}
+	}
+
+	override suspend fun getDetails(manga: Manga): Manga {
+		val doc = webClient.httpGet(manga.url.toAbsoluteUrl(domain)).parseHtml()
+		val dateFormat = SimpleDateFormat("dd/MM/yyyy", Locale.ENGLISH)
+		
+		return manga.copy(
+			altTitle = doc.selectFirst("h2.other-name")?.text(),
+			author = doc.select("p:contains(Tác Giả) + p").joinToString { it.text() },
+			tags = doc.select("a.clblue").mapNotNullToSet { 
+				MangaTag(
+					key = it.attr("href").substringAfterLast('-').substringBeforeLast('.'),
+					title = it.text(),
+					source = source,
+				)
+			},
+			description = doc.select("div.story-detail-info").text().trim(),
```

Comment:
```suggestion
			description = doc.select("div.story-detail-info").text(),
```
Consider using `.html()` instead of `.text()` -- descriptions support html tags